apps/website/src/pages/index.tsx

```diff
@@ -140,7 +144,80 @@ const getTwitchEmbed = (
   return url.toString();
 };
 
-const Home: NextPage = () => {
+// Define the schema for the expected structure of the feed entries
+const VideoSchema = z.object({
+  videoId: z.string(),
+  title: z.string(),
+  published: z.date(),
+});
+
+type Video = z.infer<typeof VideoSchema>;
+
+// Define the schema for the entire feed
+const FeedSchema = z.object({
+  feed: z.object({
+    entry: z.array(
+      z.object({
+        "yt:videoId": z.array(z.string()).nonempty(),
+        title: z.array(z.string()).nonempty(),
+        published: z.array(z.string()).nonempty(),
+      }),
+    ),
+  }),
+});
+
+type Props = {
+  videos: Video[];
+};
```

Comment:
Can we just rely on inferring the props here, like we do for our other pages? This type doesn't actually match what you're returning from static props at present, as you have an error prop sometimes returned, which inferring would handle.